libdirections-hybrid/src/main/java/com/mapbox/navigation/route/hybrid/MapboxHybridRouter.kt

```diff
@@ -75,6 +75,7 @@ class MapboxHybridRouter(
         private var reserveRouterCalled = false
         private lateinit var options: RouteOptions
         private lateinit var callback: Router.Callback
+        private var fetchingInProgress = false
 
         override fun onResponse(routes: List<DirectionsRoute>) {
             callback.onResponse(routes)
```

Comment:
I believe we should also make `fetchingInProgress` `false` when receiving a response, right?

Reply:
fixed